addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -326,12 +327,18 @@ QUnit.module('web_editor', {}, function () {
                 "should have rendered the field correctly in edit");
 
             var fontElement = $field.find('.note-editable font')[0];
-            var rangeControl = {
-                sc: fontElement,
-                so: 0,
-                ec: fontElement,
-                eo: 1,
+
+            var rangeControl = new Range();
+            rangeControl.setStart(fontElement, 0);
+            rangeControl.setEnd(fontElement, 1);
+            rangeControl = getDeepRange($field.find('.note-editable')[0], { rangeControl });
```

Comment:
I don't understand, aren't the assertions done sequentially, how can it affect what's above when it's already done.

Reply:
I meant the assertions below that test against `rangeControl`.

Reply:
There is only my assertions.

Reply:
The behavior has changed, since we do a deep range. So those test are broken now.

Reply:
That's correct but I replaced it with a simple
```js
rangeControl = {
    sc: fontElement.firstChild,
    so: 0,
    ec: fontElement.firstChild,
    eo: 9,
}
```
so as not to use `getDeepRange` and just be explicit about what we expect as a result since it's a test.